core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterDMTest.java

```diff
@@ -12522,6 +12565,20 @@ RelNode createLogicalValueRel(RexNode col1, RexNode col2) {
     assertThat(toSql(root, DatabaseProduct.ORACLE.getDialect()), isLinux(expectedOracleSql));
   }
 
+  @Test public void testForTruncTimestampFunction() {
+    final RelBuilder builder = relBuilder();
+    final RexNode truncTimestampNode = builder.call(SqlLibraryOperators.TRUNC_TIMESTAMP,
+        builder.call(CURRENT_TIMESTAMP),
+        builder.literal("Year"));
+    final RelNode root = builder
+        .scan("EMP")
+        .project(truncTimestampNode)
+        .build();
+    final String expectedBQSql = "SELECT TRUNC_TIMESTAMP(CURRENT_TIMESTAMP, 'Year') AS $f0\nFROM"
+        + " scott.EMP AS EMP";
+
+    assertThat(toSql(root, DatabaseProduct.DB2.getDialect()), isLinux(expectedBQSql));
+  }
```

Comment:
Add test case for `Concat_Ws` , `Collect_list` spark functions
and  `BitNot` for DB2

Reply:
@nosheenSheikh02 I had added the test cases.Please have a look.